### ISSUE TITLE
Headless Chrome/Firefox instead of PhantomJS for example integration

### ISSUE THREAD

**Issue by fredrikekelund** [cgiffard/node-simplecrawler#386]
The lead developer for PhantomJS is stepping down and both Chrome and Firefox are introducing a headless mode. We should look into updating the example code for how to integrate a headless browser with simplecrawler!

**Comment by MrSnoozles** [comment]
Agree, would like to see how to combine simplecrawler with a headless browser.

**Comment by gombosg** [comment]
I'm using a headless browser. Basically, after headers are fetched, in the download condition I tell simplecrawler not to download the page and instead tell my headless module to do so.

Not very reliable when downloading 10000s of pages though.
